txscript/script.go

```diff
@@ -164,6 +170,9 @@ var ErrUnsupportedAddress = errors.New("unsupported address type")
 // This timestamp corresponds to Sun Apr 1 00:00:00 UTC 2012.
 var Bip16Activation = time.Unix(1333238400, 0)
 
+// curve halforder, used to tame ECDSA malleability (see BIP0062)
+var halforder = new(big.Int).Rsh(new(big.Int).Set(btcec.S256().N), 1)
```

Comment:
The `btcec.S256().N` parameter is already a `big.Int`.  No need to create the extra one here:

var halforder = new(big.Int).Rsh(btcec.S256().N, 1)